nginx.conf

```diff
@@ -38,15 +38,26 @@ http {
   			proxy_pass http://$VulnerableApplication:80;
         }
         
-        location /VulnerabilityDefinitions {
+        location ~ ^/(?:VulnerabilityDefinitions|scanner\/sast)$ {
         	default_type 'application/json';
         	content_by_lua_block {
-        		local vulnerableAppResponse = ngx.location.capture("/VulnerableApp/VulnerabilityDefinitions")
-                local vulnerableAppJspResponse = ngx.location.capture("/VulnerableApp-jsp/VulnerabilityDefinitions")
-        		local vulnerableAppPhpResponse = ngx.location.capture("/VulnerableApp-php/VulnerabilityDefinitions")
+        		local vulnerableAppResponse = ngx.location.capture("/VulnerableApp" .. ngx.var.uri)
```

Comment:
I think it is good to have a separate configuration for VulnerabilityDefinitions and /scanner/sast, reason being later on if facade is integrated with third party then following the convention can be tricky so we may need to break this method at that point so let's break it now only.

Reply:
Okay, I can do that. I was mainly concerned with duplication in the configuration especially if the application should grow?